site/phpstan-baseline.neon

```diff
@@ -371,6 +360,26 @@ parameters:
 			count: 1
 			path: app/entities/email/EmailEntity.php
 
+		-
+			message: "#^Property app\\\\entities\\\\poll\\\\Option\\:\\:\\$id is never written, only read\\.$#"
+			count: 1
+			path: app/entities/poll/Option.php
+
+		-
+			message: "#^Property app\\\\entities\\\\poll\\\\Option\\:\\:\\$poll \\(app\\\\entities\\\\poll\\\\Poll\\) does not accept null\\.$#"
```

Comment:
Could allow for type to be null so this can go away

Reply:
Isn't this deliberate though?  We never want nulls in this DB column (and we want Doctrine to know that), but we need to be able to delete an option without deleting a poll, right?  It definitely seems like something which should be fixed, but I'm not sure how to go about doing it best.

Reply:
Good point. You could consider enabling orphanRemoval which makes the code simpler and would remove this error.

Here's an example from my course materials refactor:
https://github.com/Submitty/Submitty/blob/a882f5a032c251bf13ca358d11e24d0651d30821/site/app/entities/course/CourseMaterial.php#L66-L70
https://github.com/Submitty/Submitty/blob/a882f5a032c251bf13ca358d11e24d0651d30821/site/app/entities/course/CourseMaterial.php#L178-L180
https://github.com/Submitty/Submitty/blob/a882f5a032c251bf13ca358d11e24d0651d30821/site/app/controllers/course/CourseMaterialsController.php#L419-L423

Reply:
Good call.  Updates made.